.travis.yml

```diff
@@ -7,10 +7,18 @@ node_js:
 # Use containers.
 # http://docs.travis-ci.com/user/workers/container-based-infrastructure/
 sudo: false
+addons:
+  apt:
+    packages:
+    - qemu-utils
+    - blktap-utils
 
 before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash
   - export PATH="$HOME/.yarn/bin:$PATH"
 
 cache:
   yarn: true
+
+script:
+  - yarn run test-integration
```

Comment:
Will it still run the unit tests?

Reply:
yes, because travis runs "yarn" which implicitly runs the tests

Reply:
Nope, `yarn` does not run the tests.

Reply:
I will check then